.github/workflows/ci.yml

```diff
@@ -236,6 +236,18 @@ jobs:
       NUM_RUNNERS: ${{ matrix.num_runners }}
       RUNNER_INDEX: ${{ matrix.runner_index }}
 
+    services:
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_HOST_AUTH_METHOD: trust
```

Comment:
The docs at https://docs.github.com/en/actions/using-containerized-services/creating-postgresql-service-containersm show:
```
        # Provide the password for postgres
        env:
          POSTGRES_PASSWORD: postgres
```

Some of the fog test py scripts call `dropdb` and `createdb` and they don't expose an easy way to pass a password down. Setting to `trust` doesn't seem any worse than having a hardcoded password.